app/pages/organizations/organization-page.jsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import classnames from 'classnames';
+import { Markdown } from 'markdownz';
+import Thumbnail from '../../components/thumbnail';
+import ProjectCard from '../../partials/project-card';
+
+const AVATAR_SIZE = 100;
+
+class OrganizationPage extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      readMore: false
+    };
+  }
+
+  handleViewToggle() {
+    this.props.toggleCollaboratorView();
+  }
+
+  toggleReadMore() {
+    this.setState({ readMore: !this.state.readMore });
+  }
+
+  render() {
+    const [aboutPage] = this.props.organizationPages.filter(page => page.url_key === 'about');
+
+    const aboutContentClass = classnames(
+      'organization-details__about-content',
+      { 'organization-details__about-content--expanded': this.state.readMore });
+
+    return (
+      <div className="organization-page">
+        <section
+          className="organization-hero"
+          style={{ backgroundImage: `url(${this.props.organizationBackground.src})` }}
+        >
+          <div className="organization-hero__container">
+            {this.props.organizationAvatar &&
+              <Thumbnail
+                src={this.props.organizationAvatar.src}
+                className="avatar organization-hero__avatar"
+                width={AVATAR_SIZE}
+                height={AVATAR_SIZE}
+              />}
+            <div>
+              <h1 className="organization-hero__title">{this.props.organization.display_name}</h1>
+              <p className="organization-hero__description">{this.props.organization.description}</p>
+            </div>
+          </div>
+        </section>
+
+        <section className="resources-container">
+          {this.props.collaborator &&
+            <label className="organization-page__toggle" htmlFor="collaborator view">
+              <input onChange={this.handleViewToggle.bind(this)} type="checkbox" value={this.props.collaboratorView} />
+              <span>Collaborator View</span>
+            </label>}
+          <div className="project-card-list">
+            {this.props.organizationProjects.map(project =>
```

Comment:
We should have a UI if there are no projects linked yet and the array is empty, since we are allowing organization owners and collaborators to view the org page before it is listed.

Reply:
I think having a loading indicator and possibly an error state UI for projects separate from the org loading would be a good idea too.